andino_bringup/config/raspicam.yaml

```diff
@@ -1,7 +1,9 @@
+# Calibration tutorial (from Nav2 docs): https://docs.nav2.org/tutorials/docs/camera_calibration.html
+# Other info for this Raspberry Pi Camera Module V2 can be found here: https://elinux.org/Rpi_Camera_Module#Technical_Parameters_.28v.2_board.29
+
 image_width: 640
 image_height: 480
-# camera_name: narrow_stereo
-camera_name: hd_webcam:_hd_webcam
+camera_name: narrow_stereo
```

Comment:
nit: not sure where this name is being used however it catches my attention it says stereo when it is actually mono

Reply:
Yeah, but the name is fixed by the hardware (happens the same when I used my webcam) so there is anything we can do about that. If I change the name on the YAML it won't use that file because it doesn't have the right name.